admin/pre_reg/forms.py

```diff
@@ -36,3 +36,10 @@ class DraftRegistrationForm(forms.Form):
         ),
         required=False
     )
+
+    def __init__(self, *args, **kwargs):
+        prereg_reviewers = itertools.chain(
```

Comment:
This should fix the bug that @zkraime was running into trying to query the users table before it existed.